crates/bitwarden/src/vault/cipher/login.rs

```diff
@@ -38,6 +38,25 @@ pub struct LoginUriView {
     pub r#match: Option<UriMatchType>,
 }
 
+#[derive(Serialize, Deserialize, Debug, JsonSchema, Clone)]
+#[serde(rename_all = "camelCase", deny_unknown_fields)]
+#[cfg_attr(feature = "mobile", derive(uniffi::Record))]
+pub struct Fido2Credential {
+    pub credential_id: EncString,
+    pub key_type: EncString,
+    pub key_algorithm: EncString,
+    pub key_curve: EncString,
+    pub key_value: EncString,
+    pub rp_id: EncString,
+    pub user_handle: Option<EncString>,
+    pub user_name: Option<EncString>,
+    pub counter: EncString,
+    pub rp_name: Option<EncString>,
+    pub user_display_name: Option<EncString>,
```

Comment:
I feel like we're going to need `user_name`, `rp_name`, or `user_display_name` decrypted eventually to display them in the UI, but if it's not a need now we can leave that for the future.